ivy/functional/frontends/paddle/tensor/search.py

```diff
@@ -85,3 +85,22 @@ def masked_select(x, mask, name=None):
 )
 def topk(x, k, axis=None, largest=True, sorted=True, name=None):
     return ivy.top_k(x, k, axis=axis, largest=largest, sorted=sorted)
+
+
+# where
+@with_supported_dtypes(
+    {
+        "2.5.1 and below": (
+            "bfloat16",
+            "float16",
+            "float32",
+            "float64",
+            "int32",
+            "int64",
+        )
+    },
+    "paddle",
+)
+@to_ivy_arrays_and_back
+def where(condition, x=None, y=None, name=None):
```

Comment:
`x` and `y` are not optional in paddle ([source](https://github.com/PaddlePaddle/Paddle/blob/release/2.2/python/paddle/tensor/search.py#L473)) so they shouldn't be optional here